var/spack/repos/builtin/packages/mpas-model/package.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import llnl.util.tty as tty
+
+
+class MpasModel(MakefilePackage):
+    """The Model for Prediction Across Scales (MPAS) is a collaborative
+    project for developing atmosphere, ocean and other earth-system
+    simulation components for use in climate, regional climate and weather
+    studies."""
+
+    homepage = "https://mpas-dev.github.io/"
+    url = "https://github.com/MPAS-Dev/MPAS-Model/archive/v7.0.tar.gz"
+
+    version('7.0', sha256='f898ce257e66cff9e29320458870570e55721d16cb000de7f2cc27de7fdef14f')
+    version('6.3', sha256='e7f1d9ebfeb6ada37d42a286aaedb2e69335cbc857049dc5c5544bb51e7a8db8')
+    version('6.2', sha256='2a81825a62a468bf5c56ef9d9677aa2eb88acf78d4f996cb49a7db98b94a6b16')
+
+    variant('pnetcdf', default=True, description='use parallel netcdf')
```

Comment:
It seems like this variant isn't necessary to change the build. Users can already specify this like so:
```console
$ spack install mpas-model ^parallelio+pnetcdf
```
I would remove this variant to keep the package simple.

Reply:
fixed.